docker-compose.yml

```diff
@@ -108,3 +108,13 @@ services:
       - ./logs/api_nginx:/var/log/nginx
     ports:
       - "8055:80"
+          
+  website:
+    build: './website/client'
```

Comment:
Now that there is only a client, you should have the client directly in the website directory instead of website/client

Reply:
Yes i thought about that, will change this tonight.